extensions/interactions/base.py

```diff
@@ -91,6 +91,11 @@ class BaseInteraction(object):
     # Customization arg specifications for the component, including their
     # descriptions, schemas and default values. Overridden in subclasses.
     _customization_arg_specs = []
+    # Instructions for using this interaction, to be shown to the learner. Only
+    # relevant for supplemental interactions.
+    instructions = None
+    # Whether the answer is long, and would benefit from being summarized.
+    needs_summary = False
```

Comment:
Is there any situation when an interaction would not be inline and not require a summary? It seems this is only true for supplemental interactions.